sound/soc/sof/sof-pci-dev.c

```diff
@@ -113,13 +112,6 @@ int sof_pci_probe(struct pci_dev *pci, const struct pci_device_id *pci_id)
 	const struct snd_sof_dsp_ops *ops;
 	int ret;
 
-	if (IS_REACHABLE(CONFIG_SND_INTEL_DSP_CONFIG)) {
-		ret = snd_intel_dsp_driver_probe(pci);
-		if (ret != SND_INTEL_DSP_DRIVER_ANY && ret != SND_INTEL_DSP_DRIVER_SOF) {
-			dev_dbg(&pci->dev, "SOF PCI driver not selected, aborting probe\n");
-			return -ENODEV;
-		}
-	}
```

Comment:
Does pci-tng.c need to call snd_intel_dsp_driver_probe()?

Reply:
no. This is only required if there is an HDaudio interface, and Tangier did not have one. there's no possible conflict with the HDaudio legacy driver.